wiki/Getting-Started-UIKit.md

```diff
@@ -1,22 +1,20 @@
-# Interested in seeing examples of Workflow in action?
-Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+# Interested in seeing examples of SwiftCurrent in action?
 
-# CocoaPods with Programmatic UIKit
+Start by cloning the repo and checking out the 'SwiftCurrentExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+
+# Swift Package Manager with Programmatic UIKit
 
 ## Adding the dependency
-```ruby
-pod 'DynamicWorkflow/UIKit'
-```
-Add the above line to your Podfile.
 
-For more installation instructions, see the wiki on [installation](https://github.com/wwt/Workflow/wiki/Installation).
+For instructions on SPM and other package managers, [check out our intstallation page.](https://github.com/wwt/SwiftCurrent/wiki/Installation#swift-package-manager)
```

Comment:
This works, but I feel like it is lacking in the Best In Class level of documentation.  Specifically it requiring us to navigate around to get the information needed to get started.  It's not absurd for someone to go to the installation and we are linking directly to the section they care about, but I think we might be able to do better.

Reply:
I think so too. I wanted to include the instructions for adding the dependency via Xcode here (Ex. `File` -> `Swift Packages` -> ...), but I think @Tyler-Keith-Thompson is correct about having instructions not tied to the location in the menu. Given that, I had a tough time finding a middle ground. This is an idea:

---

To get started with SwiftCurrent using SPM, add the package using our repository url
```
https://github.com/wwt/SwiftCurrent.git
```
Other dependency approaches are covered in [our Installation wiki](https://github.com/wwt/SwiftCurrent/wiki/Installation)